ui/app/components/app/modals/new-account-modal/new-account-modal.component.js

```diff
@@ -43,6 +43,10 @@ export default class NewAccountModal extends Component {
         <div className="new-account-modal__content">
           <div className="new-account-modal__content__header">
             {t('newAccount')}
+            <div
+              className="new-account-modal__content__header-close"
+              onClick={this.props.hideModal}
+            />
```

Comment:
Could you make this a button instead of a `div`? It'd simplify the CSS for this component, and make it accessible-by-default (e.g. focus-able and responsive to keyboard input for example). Similar to the popover close button.

```suggestion
            <button
              className="fas fa-times new-account-modal__content__header-close"
              title={t('close')}
              onClick={this.props.hideModal}
            />
```